python/ql/lib/semmle/python/frameworks/Pyodbc.qll

```diff
@@ -0,0 +1,34 @@
+/**
+ * Provides classes modeling security-relevant aspects of the `pyodbc` PyPI package.
+ *
+ * See
+ * - https://github.com/mkleehammer/pyodbc/wiki
+ * - https://pypi.org/project/pyodbc/ 
+ */
+
+private import python
+private import semmle.python.dataflow.new.DataFlow
+private import semmle.python.dataflow.new.RemoteFlowSources
+private import semmle.python.Concepts
+private import semmle.python.ApiGraphs
+private import semmle.python.frameworks.PEP249
+
+/**
+ * Provides models for the `pyodbc` PyPI package.
+ *
+ * See
+ * - https://github.com/mkleehammer/pyodbc/wiki
+ * - https://pypi.org/project/pyodbc/
+ */
+private module Pyodbc {
+  // ---------------------------------------------------------------------------
+  // Pyodbc
+  // ---------------------------------------------------------------------------
```

Comment:
These are from a time where our library modeling was super verbose, and isn't really needed any longer.

```suggestion
```